airqo-platform/AirQo-api
# Map filters netsmanager api

What does this PR do?
Returns list of offline and online devices

What JIRA task is related to this PR?
No JIRA board task added for this sprint.

How do I test out this PR?
Git clone this repo cd AirQo-api/src/device-monitoring. Set up python environment: python3 -m venv env (if not yet created), then activate source env/bin/activate.
Then run pip install -r requirements/dev.txt to get all the necessary package. Afterwards, run flask run to test out the endpoints using any REST client tool available to you.
Which endpoints should be/are ready for testing?
base URL: http://localhost:4000
[GET] /api/v1/monitor/devices/online_offline
